ihmc-communication/src/main/java/us/ihmc/communication/RTPSCommunicationFactory.java

```diff
@@ -123,7 +115,6 @@ public RTPSCommunicationFactory()
       defaultAddressRestriction = foundAddressRestriction;
       if (defaultAddressRestriction != null)
          LogTools.info("Setting IP restriction: {}", defaultAddressRestriction.getHostAddress());
-      createParticipant(rtpsDomainID);
```

Comment:
A participant was getting created here I noticed when collecting stats. I don't think it's used anywhere. It seems this class was part before we had unit tests for ROS 2 and Pub Sub stuff. Now that we've got those tests. I think this stuff can get removed.